packages/fuel-indexer-tests/components/indices/simple-wasm/simple-wasm/Cargo.toml

```diff
@@ -17,3 +17,4 @@ fuels-macros = { version = "0.38.1" }
 fuels-types = { version = "0.38.1", default-features = false }
 getrandom = { version = "0.2", features = ["js"] }
 serde = { version = "1.0", default-features = false, features = ["derive"] }
+sha2 = { version = "0.10" }
```

Comment:
Same comment of all these imports, I don't think they're needed as this `sha2` is not used on the user-facing part of the indexer. If these are required for the codegen they should be made available via the plugin's prelude.